modules/ROOT/pages/edit-exchange-json-file.adoc

```diff
@@ -0,0 +1,32 @@
+= Edit your API Specification Project Exchange File
+
+Anypoint Studio enables you to edit the `exchange.json` file of your API specification project.
+
+The `exchange.json` file is a file that contains metadata required by Anypoint Exchange to describe your API asset.
+
+To edit your `exchange.json` file:
+
+. Select it from your API specification project in your Project Explorer view.
++
+image::exchange-json-package-explorer.png[alt="Exchange.json file shown in the Package explorer view"]
+. Use the editor to update the values of your `exchange.json` file:
++
+image::exchange-json-studio-editor.png[alt="Exchange.json editor view"]
++
+<1> You cannot edit the first item, *Name*, since this is the artifact name of your API specification asset.
```

Comment:
Per style guide, localization:
```suggestion
<1> You cannot edit the first item, *Name*, because this is the artifact name of your API specification asset.
```